src/platforms/flutter/index.mdx

```diff
@@ -41,8 +41,8 @@ Catch Flutter specific errors by subscribing to `FlutterError.onError`:
 
 ```dart
 FlutterError.onError = (details, {bool forceReport = false}) {
-  sentry.captureException(
-    exception: details.exception,
+  await Sentry.captureException(
+    throwable: details.exception,
```

Comment:
Is `throwable` the right name in Dart? Sounds like a Java thingy.
Given that `captureException` is the name, maybe `exception` would be ideal (even though in Dart it could be a String even)

Reply:
yeah, true, whats about just `error`? https://api.dart.dev/stable/2.10.2/dart-core/Error-class.html this is the base class, but maybe keeping `exception` one less breaking changes